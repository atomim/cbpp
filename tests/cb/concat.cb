#define JOIN(a, b) a ## b

Print "sumstuff= " + JOIN(1,2)
WaitKey
