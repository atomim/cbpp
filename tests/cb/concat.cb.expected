// #define JOIN(a, b) a ## b

Print "sumstuff= " + 12
WaitKey
